scanapp/views.py

```diff
@@ -121,3 +128,29 @@ def get(self, request, *args, **kwargs):
 
 def login(request):
     return render(request, 'scanapp/login.html')
+
+
+@csrf_exempt
+def post_sign_up(request):
```

Comment:
Use class-based views. We don't want to add csrf_exception for this view. Do we?